### PR TITLE
Modify abstract according to A&A format

### PR DESCRIPTION
**Pull request by AtreyeeS**

### CODE REVIEW

File: src/ms-review.tex [hunk 1]
```diff
@@ -190,57 +190,31 @@
...
+	data, which will require open high-level astronomy software customised for high energy astrophysics.
+
+	{Aims} In this article, we present the first the Long-term Support (LTS) release of \gammapy v1.0. \gammapy
+	is a community-developed, open-source Python package for \gammaray astronomy. Integrated into the general
+	python eco-system, it is built on Numpy, Scipy and Astropy and uses open FITS based data formats for serialisation.
+	It provides a uniform platform for reduction and modelling of data from many \gammaray instruments.
+
+	{Methods} Starting from event list and instrument response functions at the so called "Data Level 3",
+	\gammapy provides pipelines for reduction of the input data to binned WCS, HEALPix or region based data structures.
+	A variety of background reduction methods, including traditional techniques like the Reflected and Ring regions,
```
Comment (cosimoNigro) [inline src/ms-review.tex:207]: To facilitate the data reduction, several technique for background estimation are implemented in the package. 
After the data are binned, the flux of one or more gamma-ray sources can be estimated using different likelihood methods and assuming a variety of built-in spectral, temporal, and spatial models.

P.S. what is novel in the 3D field-of-view likelihood technique?

**Review by cosimoNigro — COMMENTED** [review]
Hello @AtreyeeS, I left you some comments. 
I suggest to rephrase some of the sentences.
Hope I was not too picky.

**Comment by adonath** [comment]
Thanks @AtreyeeS for the PR and thanks @cosimoNigro for the review! Do you have time to include the comments @AtreyeeS?

**Comment by AtreyeeS** [comment]
Thanks @cosimoNigro for the review! Updated accordingly

File: src/ms-review.tex [hunk 2]
```diff
@@ -190,57 +190,37 @@
...
+	{Aims} In this article, we present \gammapy, an open-source python package for the analysis of \gammaray data,
+	and illustrate the functionalities of the first long-term release of the software, version 1.0.
+	Built on the modern python scientific ecosystem, \gammapy provides a uniform platform for reducing and
+	modelling data from different gamma-ray instruments. \gammapy complies with several well-established
+	data conventions in high-energy astrophysics, providing serialised data products that are interoperable
+	with other software.
+
+	{Methods} Starting from event list and instrument response functions at the so called "Data Level 3",
+	\gammapy provides pipelines for creating reduced data binned in energy and coordinates.
+	To facilitate the rejection of the hadronic background, several techniques for background estimation
```
Comment (bkhelifi) [inline src/ms-review.tex:211]:
```suggestion
	To facilitate the handling of the hadronic background, several techniques for background estimation
```

File: src/ms-review.tex [hunk 3]
```diff
@@ -190,57 +190,37 @@
...
+	 are implemented in the package.
+	 After the data are binned, the flux and morphology of one or more \gammaray sources can be estimated
+	 using Poisson maximum likelihood fitting
+	 and assuming a variety of spectral, temporal, and spatial models.
+	  Estimation of flux points, likelihood profiles and light curves are also implemented.
+
+	{Results} After describing the structure of the package, we demonstrate the capabilities of \gammapy
+	performing several traditional analyses in gamma-ray astronomy with publicly available data:
+	spectral and spectro-morphological analyses, estimation of the light curve of a source.
+	The flexibility and power of the software is displayed in a final multi-instrument example,
```
Comment (bkhelifi) [inline src/ms-review.tex:221]:
```suggestion
	The flexibility and power of the software are displayed in a final multi-instrument example,
```

**Comment by adonath** [comment]
Thanks @AtreyeeS, I think the structured abstract is definitely an improvement over the one we had before! I'm sure the abstract will receive more comments during collaboration review, as many people consider it as one of the most important parts of a paper. But I think it is already in a very good shape. So let's merge the PR now...

**Pull request merged**